### PR TITLE
chore(aurelia): update re-exports list to match latest code

### PR DESCRIPTION
**Pull request by 3cp**
@fkleuver please especially review the exports of Scheduler/TaskQueue.

### CODE REVIEW

File: packages/aurelia/src/index.ts [hunk 1]
```diff
@@ -915,8 +962,10 @@ export {
...
-//   IDOMInitializerRegistration,
-//   DefaultComponents as RuntimeHtmlBrowserDefaultComponents,
-//   RuntimeHtmlBrowserConfiguration
-// } from '@aurelia/runtime-html-browser';
+export {
+  // IDOMInitializerRegistration,
+  // DefaultComponents as RuntimeHtmlBrowserDefaultComponents,
+  // RuntimeHtmlBrowserConfiguration
+  // BrowserDOMInitializer,
+  BrowserScheduler
```
Comment (fkleuver) [inline packages/aurelia/src/index.ts]: We shouldn't export this. `RuntimeHtmlBrowserConfiguration` takes care of it. Class is an implementation detail.

File: packages/aurelia/src/index.ts [hunk 2]
```diff
@@ -563,33 +592,53 @@ export {
...
+  // IClock,
+  // IClockSettings,
+  // IScheduler,
+  // ITask,
+  // ITaskQueue,
+  // QueueTaskOptions,
+  // Task,
+  // TaskAbortError,
+  // TaskCallback,
+  TaskQueue,
```
Comment (fkleuver) [inline packages/aurelia/src/index.ts]: Shouldn't export this. Queues are resolved via the scheduler

File: packages/aurelia/src/index.ts [hunk 3]
```diff
@@ -563,33 +592,53 @@ export {
...
 
   // ISanitizer,
   // SanitizeValueConverter,
 
   // ViewValueConverter,
 
+  // Clock,
+  // IClock,
+  // IClockSettings,
+  // IScheduler,
```
Comment (fkleuver) [inline packages/aurelia/src/index.ts]: Should export this as it's the central user api for scheduling

File: packages/aurelia/src/index.ts [hunk 4]
```diff
@@ -563,33 +592,53 @@ export {
...
+  // ITask,
+  // ITaskQueue,
+  // QueueTaskOptions,
+  // Task,
+  // TaskAbortError,
+  // TaskCallback,
+  TaskQueue,
+  // TaskQueuePriority,
```
Comment (fkleuver) [inline packages/aurelia/src/index.ts]: Should export this as it's an enum users may need for some of the scheduler apis

File: packages/aurelia/src/index.ts [hunk 5]
```diff
@@ -188,6 +186,7 @@ export {
   // InterfaceSymbol,
   InstanceProvider,
   Resolved,
+  Transformer,
```
Comment (fkleuver) [inline packages/aurelia/src/index.ts]: Don't think we need to export this actually. Users wouldn't interact with this directly.

File: packages/aurelia/src/index.ts [hunk 6]
```diff
@@ -225,6 +224,23 @@ export {
   // ReadonlyKeys,
   // WritableKeys,
 
+  // metadata,
+  // Metadata,
```
Comment (fkleuver) [inline packages/aurelia/src/index.ts]: Should export this. Metadata api is pretty important for plugin authors

File: packages/aurelia/src/index.ts [hunk 7]
```diff
@@ -225,6 +224,23 @@ export {
...
+
+  // IConsoleLike,
+  // ColorOptions,
+  // ILogConfig,
+  // ILogEvent,
+  // ILogEventFactory,
+  // ISink,
+  // ILogger,
```
Comment (fkleuver) [inline packages/aurelia/src/index.ts]: Should export this as users can inject it for logging

File: packages/aurelia/src/index.ts [hunk 8]
```diff
@@ -225,6 +224,23 @@ export {
   // ReadonlyKeys,
   // WritableKeys,
 
+  // metadata,
+  // Metadata,
+
+  // IConsoleLike,
+  // ColorOptions,
```
Comment (fkleuver) [inline packages/aurelia/src/index.ts]: Should export this as it's an enum that users may want to use in setting up logging

File: packages/aurelia/src/index.ts [hunk 9]
```diff
@@ -225,6 +224,23 @@ export {
...
+  // ILogEvent,
+  // ILogEventFactory,
+  // ISink,
+  // ILogger,
+  // LogConfig,
+  // DefaultLogEvent,
+  // DefaultLogEventFactory,
+  // DefaultLogger,
+  // ConsoleSink,
+  // LoggerConfiguration,
```
Comment (fkleuver) [inline packages/aurelia/src/index.ts]: Should export this as users need to register this in order to get logging

File: packages/aurelia/src/index.ts [hunk 10]
```diff
@@ -257,11 +276,19 @@ export {
...
   kebabCase,
+  pascalCase,
   toArray,
   // nextId,
   // resetId,
   // compareNumber,
   // mergeDistinct,
+  // isNumberOrBigInt,
+  // isStringOrDate,
+  // bound,
```
Comment (fkleuver) [inline packages/aurelia/src/index.ts]: Should export this, useful general purpose decorator

**Comment by AureliaEffect** [comment]
JS Framework Benchmark: https://34042-121594173-gh.circle-artifacts.com/0/home/circleci/repo/test/js-framework-benchmark/webdriver-ts-results/table.html

**Comment by fkleuver** [comment]
Thanks :)

**Pull request merged**